### PR TITLE
Use Alpine Linux for Docker Builds

### PR DESCRIPTION
**Pull request by gthole**
Hello! Exploring euterpe and considering the possibility of adopting it for my personal music server. I thought I'd contribute a small change in return.

I suggest switching to alpine linux in the Dockerfile.

This results in:
- Smaller image sizes
- More secure application environment
- Only use packages required for build

Thanks so much!

### CODE REVIEW

**Comment by ironsmile** [comment]
Good idea, thank you! I too prefer Alpine lately and it is only inertia which kept Debian in play here.

File: Dockerfile [hunk 1]
```diff
@@ -10,9 +9,9 @@ RUN make release
 RUN mv euterpe /tmp/euterpe
 RUN /tmp/euterpe -config-gen && sed -i 's/localhost:9996/0.0.0.0:9996/' /root/.euterpe/config.json
 
-FROM debian:buster
+FROM alpine:latest
 
-RUN apt-get update && apt-get install -y libtagc0 libicu63
+RUN apk add --update taglib-dev libc-dev icu-dev
```
Comment (ironsmile) [inline Dockerfile:14]: Aren't there non "-dev" versions of the packages? If there are I expect the image to become even smaller. I will give it a try.

Comment (ironsmile) [reply Dockerfile:14]: Strangely enough I am getting "not permitted" during building which is a complete surprise for me:

```
Step 5/15 : RUN make release
 ---> Running in 4e4bb060e991
go build \
	--tags "sqlite_icu" \
	-ldflags "-X github.com/ironsmile/euterpe/src/version.Version=`git describe --tags --always`" \
	-o euterpe
make: /bin/sh: Operation not permitted
make: *** [Makefile:9: release] Error 127
```

Comment (ironsmile) [reply Dockerfile:14]: Apparently the root problem is [this](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2). Although I don't know what to abut it :thinking: I want to be able to build the image locally on my machine.

Using the Golang 1.16-alpine-3.13 image seems to do the job.

Comment (gthole) [reply Dockerfile:14]: Yeah, I've had intermittent issues with `alpine3.14` too, so I'd be happy to switch it over to that.

There doesn't appear to be a non-dev package for `libc`, but the others do, so I'm happy to switch to those for this second `apk add` in the file.

**Comment by ironsmile** [comment]
Thank you for the suggestion! ♥ It has been added to master already with 32ce53da2c29722b64a68d6942b0a46ded16567d and 22f9aed62bf112a6d793640dce5f22729c77cde4. I am closing this PR.

On the next release the published Docker hub image will be using Alpine 🎉

**Pull request closed without merging**